src/js/binaryen.js-post.js

```diff
@@ -509,29 +524,37 @@ function initializeConstants() {
     'DanglingPop',
     'Any'
   ].forEach(name => {
-    Module['SideEffects'][name] = Module['_BinaryenSideEffect' + name]();
+    const value = MODULE['_BinaryenSideEffect' + name]();
+    SideEffects[name] = value;
+    SideEffects[value] = name;
   });
 
   // ExpressionRunner flags
-  Module['ExpressionRunner']['Flags'] = {
-    'Default': Module['_ExpressionRunnerFlagsDefault'](),
-    'PreserveSideeffects': Module['_ExpressionRunnerFlagsPreserveSideeffects'](),
-    'TraverseCalls': Module['_ExpressionRunnerFlagsTraverseCalls']()
-  };
+  [
+    'Default',
+    'PreserveSideeffects',
+    'TraverseCalls'
+  ].forEach(name => {
+    const value = MODULE['_ExpressionRunnerFlags' + name]();
+    ExpressionRunner['Flags'][name] = value;
+    ExpressionRunner['Flags'][value] = name;
+  });
 }
 
 // 'Module' interface
-Module['Module'] = function(module) {
+function BynModule(module) {
   assert(!module); // guard against incorrect old API usage
-  wrapModule(Module['_BinaryenModuleCreate'](), this);
-};
+  wrapModule(MODULE['_BinaryenModuleCreate'](), this);
+}
+
+export { BynModule as Module };
```

Comment:
A little unfortunate, but not a problem. `Module` is already defined, so we can only `export as` to expose the proper name.